app/Http/Requests/EInvoice/Peppol/CreateRequest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2024. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+namespace App\Http\Requests\EInvoice\Peppol;
+
+use App\Models\Country;
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        /**
+         * @var \App\Models\User
```

Comment:
If we encapsulate this auth logic here, we'll need to customize the response as the use will just get an authorization failure. Instead we can customize the message and throw

```php

protected function failedAuthorization()
    {
        throw new AuthorizationException($this->error_message);
    }
````